script/package.sh

```diff
@@ -51,13 +51,24 @@ fi
 
 (
 echo ""
-echo "Creating archives..."
+PLATFORM=$(uname -s)
+echo "Creating archives for $PLATFORM (${OSTYPE})..."
 mkdir output
 cd output || exit 1
-if [ "$(uname -s)" == "Darwin" ]; then
+if [ "$PLATFORM" == "Darwin" ]; then
+  echo "Using bsdtar which has some different command flags"
   tar -czf "$GZIP_FILE" -C $DESTINATION .
   tar --lzma -cf "$LZMA_FILE" -C $DESTINATION .
+elif [[ "$OSTYPE" == "msys" ]] || [[ "$OSTYPE" == "cygwin" ]]; then
```

Comment:
This might seem obscure but we're checking for Git Bash (`msys`) or Cygwin shell, and it was more reliable than OS-specific checks like `uname` (which include the OS version and architecture, making parsing tougher to do reliably).

```
$ uname
MINGW64_NT-10.0
```